DOC-12189 cbq in tools package (#3604)
couchbase/docs-server: modules/cli/pages/cli-intro.adoc

= CLI Reference

> :description: The command-line interface (CLI) tools lets you manage and monitor your Couchbase Server installation including clusters, servers, vBuckets, and XDCR.

Reviewer: subject-verb agreement

```suggestion
:description: The command-line interface (CLI) tools let you manage and monitor your Couchbase Server installation including clusters, servers, vBuckets, and XDCR.
```

:tools-ver: 7.6.2

[abstract]
{description}

The Couchbase Server installation process installs the command-line tools.

> The location of these tools depends on your platform:

Reviewer: Maybe make it clear that this is where you can find the tools after installation?

```suggestion
When you have installed Couchbase Server, you can find the command-line tools in one of the following locations, depending on your your platform:
```

[cols="50,313"]
|===
| Operating System | Directory Locations

| Linux
a| 
* [.path]`/opt/couchbase/bin`
* [.path]`/opt/couchbase/bin/install`
* [.path]`/opt/couchbase/bin/tools`

| Windows
a|
* [.path]`C:\Program Files\couchbase\server\bin`
* [.path]`C:\Program Files\couchbase\server\bin\install`

> * [.path]`C:\Program Files\couchbase\server\bin\tools`.

Reviewer: Remove period for consistency

```suggestion
* [.path]`C:\Program Files\couchbase\server\bin\tools`
```

| Mac OS X
a| 
* [.path]`/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin`
* [.path]`/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/tools`
* [.path]`/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/install`
|===

> == Managing diagnostics

Reviewer: Title case for consistency with other headings?

```suggestion
== Managing Diagnostics
```

* Windows: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-tools_{tools-ver}-windows_amd64.zip[]

Unzip or untar the packages, and the binaries are ready to run.
For example:

[source,console,subs="attributes+"]
----
wget https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-tools_{tools-ver}-linux_x86_64.tar.gz

tar -xf couchbase-server-tools_{tools-ver}-linux_x86_64.tar.gz
----

Each package also contains a `README` file and a copy of the software license.

NOTE: On Windows, you must have a recent version of the Microsoft Visual {cpp} Redistributable runtime libraries installed.
If you do not have these libraries installed, download them from https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170[Microsoft Visual {cpp} Redistributable latest supported downloads].

[#version-compatibility]
== Version Compatibility

The versions of the utilities installed by the tools package are the same as the corresponding Couchbase Server installation package.

The {tools-ver} `cbimport`, `cbexport`, `cbbackupmgr`, and `cbq` utilities are compatible with the following Couchbase Server versions:

> * 7.6.0

Reviewer 2: * 7.6.0, 7.6.1
